docs/samples/contact-center/index.html

```diff
@@ -0,0 +1,111 @@
+        </div>
+        <div>EXPAND ALL</div>
+      </div>
+    </button>
+
+    <button onclick="collapseAll()" class="global-toggle-btn" title="Collapse all">
+      <div style="display: flex; align-items: center; gap: 0.5rem;">
+        <div style="display: flex; flex-direction: column;">
+          <i class="g-arrow fa fa-angle-down" aria-hidden="true"></i>
+          <i class="g-arrow fa fa-angle-up" aria-hidden="true" style="margin-top: -8px;"></i>
+        </div>
+        <div>COLLAPSE ALL</div>
+      </div>
+    </button>
+  </div>
+
+  <!-- Webex CC / authentication-credentials -->
+  <div class="box">
+    <section class="section-box">
+      <h2 class="collapsible">
+        Auth & Registration
+        <i class="arrow fa fa-angle-up" aria-hidden="true"></i>
+      </h2>
+
+      <div class="section-content">
+        <div>
+          <select name="auth-type" id="auth-type" onchange="changeAuthType()">
+            <option value="accessToken">Access Token</option>
+          </select>
+        </div>
+
+        <fieldset>
+          <legend>Authentication</legend>
+
+          <form id="credentials">
+            <p class="note">
+              NOTE: Click <a href="https://developer.webex.com/docs/getting-started" target="_blank">here</a> for
+              access token
+            </p>
+            <div>
+              <input id="access-token" name="accessToken" placeholder="Access Token" value="" type="text">
+              <button id="access-token-save" class="btn-code" type="submit">webex.init()</button>
+              <p id="access-token-status" class="status-par">Not initialized</p>
+            </div>
+          </form>
+        </fieldset>
+      </div>
+
+       <!-- contact-center / web socket connection setup -->
+       <div>
+        <form id="webexcc-registration">
+          <fieldset>
+            <legend>Registration</legend>
+            <p class="note">
+              NOTE: Webex CC SDK websocket connection setup
+            </p>
+            <div class="u-mv">
+              <button id="webexcc-register" type="button" onclick="register()" disabled=""
+                class="btn-code">webex.cc.register()</button>
+              <p id="ws-connection-status" class="status-par">Not Subscribed</p>
+            </div>
+          </fieldset>
+        </form>
+      </div>   
+  </div>
+  </section>
+  </div>
+
+  <div class="box">
+    <section class="section-box">
+      <h2 class="collapsible">
+        Agent Desktop Using Webex CC SDK
+        <i class="arrow fa fa-angle-down" aria-hidden="true"></i>
+      </h2>
+
+      <div class="section-content">
+            </div>
+          </div>
+        </fieldset>
+      </div>
+    </section>
+  </div>
+
+  <script src="https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/handlebars@latest/dist/handlebars.js"></script>
+  <script src="../webex.min.js"></script>
+  <script src="../calling.min.js"></script>
```

Comment:
do we need these min.js files?

Reply:
We are going to use calling code for Desktop login hence I added it

Reply:
removed it now.